src/components/migrationToV2.stories.mdx

```diff
@@ -17,6 +17,12 @@ In many cases you should not need to adjust your application code at all. This w
 If you do, read on.
 
 ## Installation
+If you currently have amazon-chime-sdk-js as a dependency in your package.json, please also update it to v2.4.0 or later
```

Comment:
From @p-foucht 
> Does this example change? I thought Richard mentioned that builders can still access the transitive dependency directly

  Otherwise, I would expect our current repo demo to break since it relies on the [SDK directly for the TestSound](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/master/demo/meeting/src/utils/TestSound.tsx#L7)

Customers do not have to install `amazon-chime-sdk-js` separately. Since the component library depends on `amazon-chime-sdk-js`, they can import it without re-installing the module.

Reply:
@simmkyu is right, but also see above. It is crucial that *if* they have their own dependency, it be the same version. It's possible that we can model this with a `peerDependency` restriction to prevent an incorrect sibling, because the two versions will not be interoperable.

Reply:
Added back peerDependency.